packages/amplify_auth_cognito/example/lib/Widgets/UpdateMultipleUserAttributes.dart

```diff
@@ -0,0 +1,171 @@
+import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+import 'package:amplify_core/amplify_core.dart';
+import 'package:amplify_flutter/amplify.dart';
+import 'package:flutter/material.dart';
+
+class _UserAttributeController {
+  TextEditingController keyController;
+  TextEditingController valueController;
+  _UserAttributeController({String keyValue}) {
+    keyController = TextEditingController(text: keyValue);
+    valueController = TextEditingController();
+  }
+}
+
+// ignore_for_file: public_member_api_docs
```

Comment:
Yeah, it is a nit comment after all. Though I do think there's value in documenting example apps since they are _supposed_ to nudge users towards best practices in theory? It just seems like it would potentially be a good experience to me if I'm perusing this and the code was educational.